src/components/Button/Button.stories.tsx

```diff
@@ -6,7 +6,7 @@ import { SIZES, STATUSES } from '../ClickableStyle';
 import Icon from '../Icon';
 
 export default {
-  title: 'Components/Button',
+  title: 'Components/Button (v1)',
```

Comment:
Keeping the v1 versions of components around, so that it is easy to do comparisons and avoid snapshot change noise. Once all the component batches are completed, will remove the old versions and annotate the changes between the two.